couchbase/admin/collection/v1/collection.proto

```diff
@@ -56,10 +59,21 @@ message CreateCollectionRequest {
   string scope_name = 2;
   string collection_name = 3;
   optional uint32 max_expiry_secs = 4;
+  optional bool history_retention_enabled = 5;
 }
 
 message CreateCollectionResponse {}
 
+message UpdateCollectionRequest {
+  string bucket_name = 1;
+  string scope_name = 2;
+  string collection_name = 3;
+  optional uint32 max_expiry_secs = 4;
+  optional bool history_retention_enabled = 5;
+}
+
+message UpdateCollectionResponse {}
```

Comment:
Do we want the manifest id in here like the delete and create responses? Sorry for the bitty reviews, I'm realising things as I work through ING-656

Reply:
You should look at whether the server returns the manifest id during an update (I suspect it does).  We should add it in our response if the server provides it, since it's useful for various 'wait for server' things with memcached.

Reply:
ns_server does return the ManifestId so we'll need to add that here